src/test/ts/commands/PublishTest.ts

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I'd very much prefer that as a disable-next-line thing, or removed if we can do it

Reply:
Thanks. Yeah, it's not needed.

Reply:
Fixed.